pyccel/ast/builtins.py

```diff
@@ -402,7 +402,7 @@ def arg(self):
         return self._args[0]
 
 #==============================================================================
-class PythonList(Tuple, PyccelAstNode):
+class PythonList(PythonTuple, PyccelAstNode):
```

Comment:
Is `PythonTuple` not a subclass of `PyccelAstNode`?

Reply:
You're right